packages/metro/src/Bundler.js

```diff
@@ -17,12 +17,16 @@ import type {TransformOptions} from './DeltaBundler/Worker';
 import type {TransformResultWithSource} from './DeltaBundler';
 import type {ConfigT} from 'metro-config/src/configTypes.flow';
 
+export type BundlerOptions = {|
```

Comment:
Could you add `$ReadOnly`?

Reply:
Good call. I applied that to a couple of other places where I was doing something similar.